pkg/xds/envoy/listeners/v3/outbound_listener_configurer.go

```diff
@@ -17,7 +20,7 @@ func (c *OutboundListenerConfigurer) Configure(l *envoy_api.Listener) error {
 	l.Address = &envoy_core.Address{
 		Address: &envoy_core.Address_SocketAddress{
 			SocketAddress: &envoy_core.SocketAddress{
-				Protocol: envoy_core.SocketAddress_TCP,
+				Protocol: envoy_core.SocketAddress_Protocol(c.Protocol),
```

Comment:
I see in some tests the protocol is not provided, does it default to TCP?

Reply:
Yes as it's what you want most of the time the enum 0 is TCP

Reply:
got it, thank you :-)